torch/_torch_docs.py

```diff
@@ -2162,6 +2162,32 @@ def merge_dicts(*dicts):
     tensor(7)
 """)
 
+add_docstr(torch.vdot,
+           r"""
+vdot(x, y) -> Tensor
+
+Computes the dot product (inner product) of two tensors.
```

Comment:
What about something like:

"Computes the dot product (inner product) of input the complex conjugate of other."

And not having to explain in a note how vdot is distinct from dot?

Reply:
I think we should include the note in the description because it brings more clarity to the description.

```
"Computes the dot product (inner product) of two tensors. The vdot(a, b) function handles complex numbers differently than dot(a, b). If the first argument is complex the complex conjugate of the first argument is used for the calculation of the dot product."
```

Reply:
May still need to inline this note per @anjali411's feedback. I'll let her have final say over doc string.

Reply:
@kshitij12345 nit -- can you change the (new line) formatting to:
```
"Computes the dot product (inner product) of two tensors. The vdot(a, b) function handles complex numbers differently than dot(a, b). If the first argument is complex the complex conjugate of the first argument is used for the calculation of the dot product."
```

Reply:
Done. Thanks!